monkey/monkey_island/cc/ui/src/components/pages/ConfigurePage.js

```diff
@@ -110,6 +115,20 @@ class ConfigurePageComponent extends AuthComponent {
       });
   };
 
+  removePBAfiles(){
+    // We need to clean files from widget, local state and configuration (to sync with bac end)
+    if (this.hasOwnProperty('PBAlinuxPond') && this.PBAwindowsPond !== null){
```

Comment:
1. You mean remove?
3. This is used to check if the components are initialized (are we removing files while filepond components are rendered). Logically they can be kept together, because they are rendered on the same page, so if one is rendered, another is too. I can separate them for readability though